src/main/java/wolox/training/models/Book.java

```diff
@@ -26,25 +56,52 @@
 
     @Column(nullable = false)
     private String author;
+    public void setAuthor(String author){
+        this.author = checkString(author, "author");
+    }
 
     @Column(nullable = false)
     private String image;
+    public void setImage(String image){
+        this.image = checkString(image, "image");
+    }
 
     @Column(nullable = false)
     private String title;
+    public void setTitle(String title){
+        this.title = checkString(title, "title");
+    }
 
     @Column(nullable = false)
     private String subtitle;
+    public void setSubtitle(String subtitle){
```

Comment:
I think this setter could be private, because it's not used from another class (This way we could make this object even more immutable 😃 )